README.md

```diff
@@ -1,17 +1,12 @@
-TEST
-
 # Carbon Design System
 
 ![Deployment status](https://github.com/carbon-design-system/carbon-website/workflows/Deployment%20status/badge.svg)
 
-This is the [Carbon Design System website](http://www.carbondesignsystem.com).
-It's built using the
-[gatsby-theme-carbon](https://gatsby.carbondesignsystem.com/) with
-[GatsbyJS](https://www.gatsbyjs.org/).
+This is the [Carbon Design System website](http://www.carbondesignsystem.com).  
+It's built using the [gatsby-theme-carbon](https://gatsby.carbondesignsystem.com/) with [GatsbyJS](https://www.gatsbyjs.org/).
 
 ## 📂 Structure
 
-```
 src
 ├── components
```

Comment:
removing the backtics cause this to not be formatted properly. I'd just leave them in.